### PR TITLE
Seg3D DMG on MacOSX

### PR DESCRIPTION
**Pull request by allywarner**
New Seg3D DMG script.

### CODE REVIEW

**Comment by jessdtate** [comment]
@jcfr

**Comment by dcwhite** [comment]
@allywarner Please remove all the DS_Store files from this, I added a gitignore for them

**Comment by jcfr** [comment]
Nitpick:

Merging master back into your branch makes history hard to follow and makes eventual regression hunting challenging.

Instead, I would suggest to either:

- rebate locally, squash and force push
Or
- rebase + squash using the github interface
